src/frontends/tensorflow_common/src/op/reverse_sequence.cpp

```diff
@@ -14,14 +16,29 @@ namespace frontend {
 namespace tensorflow {
 namespace op {
 OutputVector translate_reverse_sequence_op(const NodeContext& node) {
-    default_op_checks(node, 2, {"ReverseSequence"});
+    default_op_checks(node, 2, {"ReverseSequence"}, true);
     auto input = node.get_input(0);
     auto seq_lengths = node.get_input(1);
 
     // retrieve attributes
     auto seq_dim = node.get_attribute<int64_t>("seq_dim");
     auto batch_dim = node.get_attribute<int64_t>("batch_dim", 0);
 
+    auto complex_type_mark = as_type_ptr<ComplexTypeMark>(input.get_node_shared_ptr());
+    if (complex_type_mark) {
+        auto const_one = make_shared<v0::Constant>(element::i32, Shape{1}, 1);
+        auto updated_seq_dim = make_shared<v1::Add>(seq_dim, const_one);
+        auto updated_batch_dim = make_shared<v1::Add>(batch_dim, const_one);
```

Comment:
not clear why are `seq_dim` and `batch_dim` incremented. For non-negative values, there is no update. For negative values, we should add `rank-1`, where `rank` is a rank of `input`

Please add tests with negative axes.

Reply:
Not correct because `seq_dim` is of `int64_t` type and you pass it into Graph operation `Add`. You should update it using `Select` and `Rank` operations